src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -10,7 +17,12 @@ public class AuthenticationService {
      * @return true if user by email exists and passed password is equal to user's password.
      * Return false in any other cases.
      */
+
     public boolean login(String email, String password) {
-        return false;
+        User user = userService.findByEmail(email);
+
+        //return user != null && user.getPassword().equals(password) ? true : false;
+
```

Comment:
Great, all that is left is to remove the commented code and empty lines, because usually it is not needed anymore and just makes it less readable